### DISCUSSION TITLE
有辦法讓怪物名字不漢化嗎?

### DISCUSSION THREAD

**Discussion by tenchi6707** [GpointChen/FFXIVChnTextPatch-GP#290] [Q&A]
您好
因ACT部分功能僅套用日文名字，若漢化ACT就失效了，請問有辦法不漢化怪物的名字嗎? 謝謝!

**Comment by GpointChen** [comment]
理論上清空或刪除BNpcName.csv後再漢化即可

記得要先還原

**Reply by ziegfeld** [reply]
找到了這個帖子，跟我剛剛file的 iinact cact raidboss timeline 的issue相關，
我也測試一下再過來報告！

**Comment by ziegfeld** [comment]
@tenchi6707  我也試了下，發現act raidboss timeline只是按照上面說明取消NPC名字漢化的話還是無法正常工作。請問你的問題解決了嗎？是act哪個部件呢？

**Reply by Liteya** [reply]
你要用cactbot 的話 要把logmessage 都套進去原始檔  且更改語言要重新啟動ACT
